ci: error obtaining VCS status: exit status 128

## Bug Report

### 1. Describe the bug

<!-- A clear and concise description of what the bug is. -->

CI failed as following (see https://do.pingcap.net/jenkins/blue/organizations/jenkins/tikv%2Fmigration%2Fpull_unit_test/detail/pull_unit_test/95/pipeline/):

```
error obtaining VCS status: exit status 128
	Use -buildvcs=false to disable VCS stamping.
error obtaining VCS status: exit status 128
	Use -buildvcs=false to disable VCS stamping.
error obtaining VCS status: exit status 128
	Use -buildvcs=false to disable VCS stamping.
error obtaining VCS status: exit status 128
	Use -buildvcs=false to disable VCS stamping.
error obtaining VCS status: exit status 128
	Use -buildvcs=false to disable VCS stamping.
error obtaining VCS status: exit status 128
	Use -buildvcs=false to disable VCS stamping.
error obtaining VCS status: exit status 128
	Use -buildvcs=false to disable VCS stamping.
error obtaining VCS status: exit status 128
	Use -buildvcs=false to disable VCS stamping.
no Go files in /home/jenkins/agent/workspace/tikv/migration/pull_unit_test/migration/cdc

=== Errors
no Go files in /home/jenkins/agent/workspace/tikv/migration/pull_unit_test/migration/cdc
DONE 0 tests, 1 error in 0.014s

error obtaining VCS status: exit status 128
	Use -buildvcs=false to disable VCS stamping.
error obtaining VCS status: exit status 128
	Use -buildvcs=false to disable VCS stamping.
error obtaining VCS status: exit status 128
	Use -buildvcs=false to disable VCS stamping.
error obtaining VCS status: exit status 128
	Use -buildvcs=false to disable VCS stamping.
make: *** [unit_test_in_verify_ci] Error 1
```

### 2. Minimal reproduce step (Required)

<!-- a step by step guide for reproducing the bug. -->

Run CI.

### 3. What did you see instead (Required)

CI failed.

### 4. What did you expect to see? (Required)

CI succeed.

### 5. What is your migration tool and TiKV version? (Required)

- TiKV:
- TiKV CDC: main (95dc4430a344d2f22fd6b6966fd530f1364872ad)
- TiKV BR:
- TiKV Online Bulk Load:

--- a/cdc/Makefile
+++ b/cdc/Makefile
@@ -117,11 +117,11 @@ unit_test: check_failpoint_ctl
 	|| { $(FAILPOINT_DISABLE); exit 1; }
 	$(FAILPOINT_DISABLE)
 
-unit_test_in_verify_ci: check_failpoint_ctl tools/bin/gotestsum gocov gocov-xml
+unit_test_in_verify_ci: check_failpoint_ctl gotestsum gocov gocov-xml
 	mkdir -p "$(TEST_DIR)"
 	$(FAILPOINT_ENABLE)
 	@export log_level=error;\
-	CGO_ENABLED=1 tools/bin/gotestsum --junitfile cdc-junit-report.xml -- -v -timeout 30m -p $(P) --race \
+	CGO_ENABLED=1 gotestsum --junitfile cdc-junit-report.xml -- -v -timeout 30m -p $(P) --race \
 	-covermode=atomic -coverprofile="$(TEST_DIR)/cov.unit.out" $(PACKAGES) \
 	|| { $(FAILPOINT_DISABLE); exit 1; }
 	gocov convert "$(TEST_DIR)/cov.unit.out" | gocov-xml > cdc-coverage.xml
@@ -231,8 +231,8 @@ gocov:
 gocov-xml:
 	$(GO) install github.com/AlekSi/gocov-xml@latest
 
-tools/bin/gotestsum: tools/check/go.mod
-	cd tools/check && $(GO) build -mod=mod -o ../bin/gotestsum gotest.tools/gotestsum
+gotestsum:
+	$(GO) install gotest.tools/gotestsum@latest
 
 tools/bin/errdoc-gen: tools/check/go.mod
 	cd tools/check && $(GO) build -mod=mod -o ../bin/errdoc-gen github.com/pingcap/errors/errdoc-gen

